packages/thirdweb/src/react/core/hooks/auth/useSiweAuth.ts

```diff
@@ -96,11 +96,11 @@ export function useSiweAuth(authOptions?: SiweAuthOptions) {
     requiresAuth,
 
     // login
-    doLogin: loginMutation.mutate,
+    doLogin: loginMutation.mutateAsync,
```

Comment:
why the switch to `async`? mutate catches any async errors for us which is useful here, no?